src/JustSaying/Fluent/QueueSubscriptionBuilder`1.cs

```diff
@@ -171,8 +172,7 @@ public QueueSubscriptionBuilder<T> WithTag(string key, string value)
             subscriptionConfig.Validate();
 
             var queue = creator.EnsureQueueExists(config.Region, subscriptionConfig);
-            bus.AddStartupTask(
-                queue.Queue.EnsureQueueAndErrorQueueExistAndAllAttributesAreUpdatedAsync(subscriptionConfig));
+            bus.AddStartupTask(queue.StartupTask);
```

Comment:
So it turns out we were calling this method (`EnsureQueueAndErrorQueueExistAndAllAttributesAreUpdatedAsync`, what a mouth full!) and setting the `StartupTask` to resulting task, that might have resulted in weird behavior as we are running it twice in 2 different places and awaiting one of them.

This means that `IVerifyAmazonQueues` no longer needs to be generic, and the everything can be internalized. @gkinsman can you double-check my thinking here.

Reply:
Ahh, good catch! This is unintentional. Thinking about it again now, I'm not sure we should be starting the task in AmazonQueueCreator as it will start running the task in the background, when we really want to start running it when the bus is started. 

The intention here was for it to be a cold task, but I think I've been writing too much JS recently as C# doesn't do that 😅. We might need to change the startup task to be a `Func<Task>`.